controllers/nginx/pkg/cmd/controller/nginx.go

```diff
@@ -256,7 +257,12 @@ func (n NGINXController) Info() *ingress.BackendInfo {
 
 // OverrideFlags customize NGINX controller flags
 func (n NGINXController) OverrideFlags(flags *pflag.FlagSet) {
-	flags.Set("ingress-class", "nginx")
+	flags.Set("ingress-class", defIngressClass)
```

Comment:
This is the same as current behaviour, I just moved to a variable.
Ingress-class will be configured on https://github.com/gianrubio/ingress/blob/2ddba72baa451abffd036de4306a91062e82a146/core/pkg/ingress/controller/launch.go#L146